src/tests/simple_dfs.c

```diff
@@ -0,0 +1,101 @@
+/**
+ * (C) Copyright 2020-2023 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+
+/*
+ * This provides a simple example for how to get started with a DFS container.
+ */
+
+#include <stdlib.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+
+#include <daos.h>
+#include <daos_fs.h>
+
+#define FAIL(fmt, ...)						\
+do {								\
+	fprintf(stderr, fmt " aborting\n", ## __VA_ARGS__);	\
+	exit(1);						\
+} while (0)
+
+#define	ASSERT(cond, ...)					\
+do {								\
+	if (!(cond))						\
+		FAIL(__VA_ARGS__);				\
+} while (0)
+
+
+int
+main(int argc, char **argv)
+{
+	dfs_t		*dfs;
+	dfs_obj_t	*dir1, *f1;
+	int		rc;
+
+	if (argc != 3) {
+		fprintf(stderr, "usage: ./exec pool cont\n");
```

Comment:
`./exec`? Should that be replaced with the real name?

Reply:
executable name which can be anything. while this is built with scons today, the main use for this is just someone downloading the test program, compiling it themselves and running it (the binary for this is not released with rpms - same as the other examples in this dir).